text/0000-optimize-by-default.md

```diff
@@ -0,0 +1,166 @@
+- Start Date: 2015-03-10
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Make rustc and cargo produce optimized binaries by default.
+
+# Motivation
+
+
+Optimizing by default is a safety & usability feature that everyone benefits
```

Comment:
Thank you for clarifying.

Prescriptively quoting dictionary definitions isn't so helpful. Descriptively, the Rust community has connotations around "safety" (e.g. the `unsafe` keyword is not written `memory_unsafe`) beyond the straight definition: it has kind-of become a piece of technical jargon in Rust. You may agree or not with this, but still, that's the way it is and ignoring it leads to imprecision and possible confusion. (My initial comment was just trying to avoid this by ensuring that everyone is on the same page.)